nux/store/selectors.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Returns whether the tip has already been shown
+ *
+ * @param  {Object}  state Global application state
+ * @param  {String}  tipId The tip ID
+ * @return {Boolean}       Whether the tip has been shown or not
+ */
+export function tipHasBeenShown( state, tipId ) {
+	return state.nux.tips[ tipId ];
```

Comment:
Any chance this would return `undefined`?

Reply:
good point, I think it's possible which still produces a "falsy" value but might better to force `false`